FEXCore/Source/Interface/Core/OpcodeDispatcher.h

```diff
@@ -1171,7 +1171,7 @@ friend class FEXCore::IR::PassManager;
     if (SetBits == 0)
       return _Lshl(Value, _Constant(Bit));
     else if (CTX->BackendFeatures.SupportsShiftedBitwise && (SetBits & (1u << Bit)) == 0)
-      return _Orlshl(NZCV, Value, Bit);
+      return _Orlshl(IR::SizeToOpSize(std::max(GetOpSize(NZCV), GetOpSize(Value))), NZCV, Value, Bit);
```

Comment:
This is intended to always be 32-bit operation but it might not matter in practice.

Reply:
Yea, I noticed that but I'm leaving it alone for now since I don't want codegen to change with these right now. Tackling one problem at a time.
Changing it to 32-bit only operation causes InstCountCI churn half-way through this process but we can make sure to come back to this once implicit size is gone.

Reply:
https://github.com/FEX-Emu/FEX/issues/3019 To ensure we don't forget

Reply:
Seems reasonable, thx